# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sysvinit/autocon.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- sysvinit-3.00/src/init.c.vanilla	2021-12-01 11:26:25.620000256 +0100
+++ sysvinit-3.00/src/init.c	2021-12-01 11:28:49.812000325 +0100
@@ -1366,7 +1366,7 @@
 	}    
 	if (fgets(buf, sizeof(buf), fp)) {    
 		char* p = buf;
-           if ( strstr(p, "init.autocon=1") )
+           if (!strstr(p, "init.autocon=0"))
            {
 		while ((p = strstr(p, "console="))) {    
 			char* e;
